cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -156,26 +156,24 @@ func SetDefaults_NodeConfiguration(obj *NodeConfiguration) {
 	}
 }
 
-// SetDefaultsEtcdSelfHosted sets defaults for self-hosted etcd
+// SetDefaultsEtcdSelfHosted sets defaults for self-hosted etcd if used
```

Comment:
Why would we ever have selfhosted etcd?

Reply:
This is operator-related. It was added in https://github.com/kubernetes/kubernetes/pull/54543 to break up my monolith PR.

Reply:
this fix just fixes the bug where this was filled in automatically although it shouldn't have been